applications/RANSApplication/custom_processes/rans_apply_exact_nodal_periodic_condition_process.h

```diff
@@ -0,0 +1,150 @@
+//    |  /           |
+//    ' /   __| _` | __|  _ \   __|
+//    . \  |   (   | |   (   |\__ `
+//   _|\_\_|  \__,_|\__|\___/ ____/
+//                   Multi-Physics
+//
+//  License:         BSD License
+//                   Kratos default license: kratos/license.txt
+//
+//  Main authors:    Suneth Warnakulasuriya
+//
+
+#if !defined(KRATOS_RANS_APPLY_EXACT_NODAL_PERIODIC_CONDITION_PROCESS_H_INCLUDED)
+#define KRATOS_RANS_APPLY_EXACT_NODAL_PERIODIC_CONDITION_PROCESS_H_INCLUDED
+
+// System includes
+#include <string>
+#include <vector>
+
+// External includes
+
+// Project includes
+#include "containers/model.h"
+#include "processes/process.h"
+
+namespace Kratos
+{
+///@addtogroup RANSApplication
+///@{
+
+///@name Kratos Classes
+///@{
+
+/**
+ * @brief A process to create periodic conditions between two boundaries
+ *
+ * This process does node to node matching to create periodic boundary
+ * conditions between the master and slave model parts, to be used in ResidualBasedBlockBuilderAndSolverPeriodic.
+ *
+ * This process requires two boundaries to have maching meshes.
+ *
+ * @see class ResidualBasedBlockBuilderAndSolverPeriodic
+ */
+class KRATOS_API(RANS_APPLICATION) RansApplyExactNodalPeriodicConditionProcess : public Process
+{
+public:
+    ///@name Type Definitions
+    ///@{
+
+    /// Pointer definition of RansApplyExactNodalPeriodicConditionProcess
+    KRATOS_CLASS_POINTER_DEFINITION(RansApplyExactNodalPeriodicConditionProcess);
+
+    using NodeType = ModelPart::NodeType;
```

Comment:
I think you are not using this type definition.

Reply:
removed :)